tests/storage/datavolume.go

```diff
@@ -415,30 +402,8 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 					return err
 				}, 30*time.Second, 1*time.Second).Should(BeNil())
 
-				vmi := tests.NewRandomVMI()
-
-				diskName := "disk0"
-				vmi.Spec.Domain.Devices.Disks = append(vmi.Spec.Domain.Devices.Disks, v1.Disk{
-					Name: diskName,
-					DiskDevice: v1.DiskDevice{
-						Disk: &v1.DiskTarget{
-							Bus: v1.DiskBusVirtio,
-						},
-					},
-				})
-				vmi.Spec.Volumes = append(vmi.Spec.Volumes, v1.Volume{
-					Name: diskName,
-					VolumeSource: v1.VolumeSource{
-						PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{PersistentVolumeClaimVolumeSource: k8sv1.PersistentVolumeClaimVolumeSource{
-							ClaimName: dv.ObjectMeta.Name,
-						}},
-					},
-				})
-
-				vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("512M")
-
 				vm := tests.NewRandomVirtualMachine(vmi, true)
-				_, err = virtClient.VirtualMachine(util.NamespaceTestDefault).Create(vm)
+				vm, err = virtClient.VirtualMachine(util.NamespaceTestDefault).Create(vm)
```

Comment:
@Barakmor1 I can't see where do you use `vm`? since you've changed `_, err` to `vm, err`

Reply:
Hey @enp0s3 ,
Thanks for reviewing
The reason i added this is to fill the `vm.namespace` field because 
we use it in line 410 (`eventually` scope).

Reply:
@Barakmor1 Thanks!